client/dashboard/utils/site-owner.ts

```diff
@@ -1,5 +1,5 @@
 import type { Site } from '../data/types';
 
 export function isA8CSite( site: Site ) {
-	return site.site_owner === 26957695;
+	return site.is_a8c;
```

Comment:
Could we move this inline? I moved this to a util because of the number check, but that's gone now.

Reply:
Good idea. I have moved the check inline and removed the `isA8CSite` utility.